cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/notification/RetrievalNotifier.java

```diff
@@ -0,0 +1,10 @@
+package com.sequenceiq.cloudbreak.cloud.notification;
+
+import java.util.List;
+
+import com.sequenceiq.cloudbreak.cloud.model.CloudResource;
+
+public interface RetrievalNotifier {
+
+    List<CloudResource> notifyRetrieveByResourceReference(String resourceReference);
```

Comment:
Yes, because every stack creates a new managed image record into the resource table. But, I think it's enough to know that is there any image in REQUESTED status.